packages/kernel-6.1/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "kernel-6_1"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[package.metadata.build-package]
+variant-sensitive = "platform"
+package-name = "kernel-6.1"
+
+[lib]
+path = "pkg.rs"
```

Assuming #3148 merges first:
```suggestion
build = "../build.rs"

[package.metadata.build-package]
variant-sensitive = "platform"
package-name = "kernel-6.1"

[lib]
path = "/dev/null"
```

(And then remove `pkg.rs`).